apps/climatemappedafrica/src/components/HURUmap/Tutorial/index.js

```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles(({ typography, palette }) => ({
   },
 }));
 
-function Tutorial({ children, defaultOpen, items, ...props }) {
+function Tutorial({ children, defaultOpen = false, items = [], ...props }) {
```

Comment:
Changing items from `undefined` to `[]`?

Reply:
I was getting ahead of myself thinking, after looking into propTypes, but I see it is unnecessary since we use optional chaining as we access items 👍